WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUp.swift

```diff
@@ -1,7 +1,7 @@
 import SwiftUI
 
-struct InPersonPaymentsCodPaymentGatewayNotSetUp: View {
-    let viewModel: InPersonPaymentsCodPaymentGatewayNotSetUpViewModel
+struct InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUp: View {
```

Comment:
Should we rename the struct and file to `InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpView` ?

Also, perhaps we should move the `cashOnDeliveryLearnMoreUrl` URL from the `Constants` enum to `WooConstants.URL`?

Reply:
> Should we rename the struct and file to `InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpView` ?

Yep, most of the others do have the View suffix and I prefer that generally, so I'll do that.


> Also, perhaps we should move the `cashOnDeliveryLearnMoreUrl` URL from the `Constants` enum to `WooConstants.URL`?

Hmm, good call. I like it in Constants because it's nice and close to the call site so it makes it a bit more obvious what's going on, and it won't be used elsewhere. 

But. 

It looks like WooConstants.URL implies a degree of trust in the URL, and it's a CaseIterable Enum, so it's probably used (or could be used) to lock a web view down to only be able to access specific URLs, which is an option with WKWebView. I think it's worth making sure this is on that list.